data-processing/entities/definitions/model/load_data.py

```diff
@@ -555,6 +555,80 @@ def load_and_cache_example(
     )
     return dataset
 
+def load_and_cache_example_batch_raw(
+    args: Any, tokenizer: AutoTokenizer, data: List[Dict[Any, Any]]
+) -> TensorDataset:
+    processor = DefProcessor(args)
+
+    examples = []
+    raw = []
+    for d in data:
+        example = InputExample(
+            guid="one",
+            words=d["tokens"],
+            intent_label=1,  # fakse intent label
+            slot_labels=[1] * len(d["tokens"]),  # fake slot labels
```

Comment:
That is correct, these are fake labels (used for prediction when no labels exists, I think). I had this doubt too, but I think this is correct.

Reply:
Let us give the fake labels a value that more accurately demonstrates that the data is invalid (e.g., -1 if it can be an int, or 'unknown' if a string)

Reply:
I'm using -1 for this

Reply:
Actually I cannot use -1 😱 
It triggers a CUDA error. The intent label has to be one of the 2 possible values (0 or 1), and the slot label has to be one of 6 possible values (0 to 5)